java/junit5/pom.xml

```diff
@@ -61,6 +61,12 @@
             <artifactId>sauce_bindings</artifactId>
             <version>1.4.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.28</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
I only saw usage in two parts. Could we remove this dependency?

Reply:
If we're going to use it in one part of the monorepo, I think the methods should be available everywhere

Reply:
Fair enough.